django_prometheus/middleware.py

```diff
@@ -39,11 +49,16 @@ def process_response(self, request, response):
             requests_unknown_latency_before.inc()
         return response
 
+if LATENCY_BUCKETS is not None:
+    requests_latency_by_view_method = Histogram(
+        'django_http_requests_latency_seconds_by_view_method',
+        'Histogram of request processing time labelled by view.',
+        ['view', 'method'], buckets=LATENCY_BUCKETS)
+else:
+    requests_latency_by_view_method = Histogram(
+        'django_http_requests_latency_seconds_by_view_method',
+        'Histogram of request processing time labelled by view.', ['view', 'method'])
```

Comment:
This line is not passing pep8, so Travis is choking. Please fix

Reply:
Done.